ansible_collections/arista/avd/molecule/eos_designs_unit_tests/inventory/host_vars/snmp-settings-2.yml

```diff
@@ -0,0 +1,44 @@
+---
+# Minimum config to only test the specific feature.
+# This tests the VRFs and SNMP hosts only using inband management VRF.
+type: l2leaf
+
+l2leaf:
+  nodes:
+    - name: snmp-settings-2
+      id: 43
+      inband_mgmt_ip: 192.168.0.1/24
+      inband_mgmt_interface: Vlan123
+      # inband_mgmt_vrf: default ("default" by default)
+
+snmp_settings:
+  enable_inband_mgmt_vrf: true # This one *should* be in the output, since this host *has* inband mgmt
+  enable_mgmt_interface_vrf: true # This one should not be in the output, since this host has no inband mgmt
```

Comment:
is this comment correct? it seems to indicate that the host has no inband mgmt but the previous line says the opposite

(same below)

Reply:
Fixed. Thanks